stories/object-attribute/object-attribute.stories.js

```diff
@@ -0,0 +1,27 @@
+export default {
+    title: 'Dev/Object Attribute',
+    components: ['object-attribute']
+};
+
+export const Text = () => `<div class="fd-object-attribute" id="standardtext">Standard Object Attribute Text</div>
+`;
+
+Text.parameters = { docs: { disable: true } };
+
+export const TextWithLink = () => `<div id="firsttext" class="fd-object-attribute">Object Attribute :
+<span id="textwithlink" class="fd-object-attribute--link" role="link" tabindex="0">Text With Link</span></div>
```

Comment:
For the sake of semantics, all links should be an `<a>` with proper href. If we want generic actions an example for a button should be added.